Dockerfile

```diff
@@ -57,18 +57,23 @@ RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=$SCCACHE_DIR,sharing=locked \
     cargo build --release --features="$FEATURES"
 
+FROM ethpandaops/reth:main-d6113e1 as reth
```

Comment:
@parithosh any reason you used a specific version for reth? Not just latest?

Reply:
uhh none actually, feel free to change

Reply:
just wanted to use a known working version